uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -138,6 +139,34 @@ define([
       watchFieldAllowCarryForward();
     }
 
+    function initValidatorForField (field) {
+      $scope.$watch(function () {
```

Comment:
`$scope.$watch(field.value, function (value) { ... } )`
Will this work? then we can get rid of one function.

Reply:
No, it does not work.